src/ripple/app/ledger/LedgerProposal.cpp

```diff
@@ -43,21 +45,25 @@ LedgerProposal::LedgerProposal (
     , mProposeSeq (seq)
     , publicKey_ (publicKey)
     , mPeerID (nodeID)
-    , mTime (std::chrono::steady_clock::now ())
+    , mTime (now)
 {
+    signature_.resize (signature.size());
+    std::memcpy(signature_.data(),
+        signature.data(), signature.size());
```

Comment:
`signature_` should be a `Buffer`, and `Buffer` should support `operator=(Slice const&)`